### PR TITLE
Feature/DO-595 Organisation Update Endpoint

### PR DESCRIPTION
**Pull request by shilpigoel-goaco**

### CODE REVIEW

**Review by rmohammed-goaco — APPROVED** [review]

File: Services/CO.CDP.Organisation.WebApi/Program.cs [hunk 1]
```diff
@@ -83,9 +83,9 @@
 builder.Services.AddOrganisationProblemDetails();
 
 builder.Services.AddJwtBearerAndApiKeyAuthentication(builder.Configuration, builder.Environment);
-//builder.Services.AddAuthorization();
-builder.Services.AddOrganisationAuthorization();
-builder.Services.AddGovUKNotifyApiClient(builder.Configuration);
+builder.Services.AddAuthorization();
+//builder.Services.AddOrganisationAuthorization();
+//builder.Services.AddGovUKNotifyApiClient(builder.Configuration);
```
Comment (andymantell) [inline Services/CO.CDP.Organisation.WebApi/Program.cs:88]: Probably want to revert these changes before they go to master

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Services/CO.CDP.Organisation.WebApi/Program.cs (hunk 1 above)
Comment (shilpigoel-goaco) [reply Services/CO.CDP.Organisation.WebApi/Program.cs]: good spotted, resolved

**Review by andymantell — APPROVED** [review]
